src/hooks/useSaveCursorPosition.tsx

```diff
@@ -12,7 +14,7 @@ export default function useSaveCursorPosition(
 
     React.useEffect(() => {
         const inputElement: HTMLInputElement = inputRef.current;
-        if (inputElement) {
+        if (inputElement && !NOT_ALLOWED_SELECTION_TYPES.includes(inputElement.type)) {
```

Comment:
Такая проверка чревата тем, что в браузерах может поменяться поддержка setSelectionRange, а мы об этом не узнаем.

Предлагаю делать проверку на само наличие этого метода в объекте, типа
```suggestion
        if (typeof inputElement?.setSelectionRange === 'function') {
```

Reply:
Какой бы мы тип не передали эта проверка всегда будет выполняться.